plugins/polio/tests/test_base.py

```diff
@@ -683,6 +686,181 @@ def test_create_calendar_xlsx_sheet_with_separate_scopes_per_round(self):
         self.assertEqual(data_dict["January"][0], self.format_date_to_test(c, c_round_1))
         self.assertEqual(data_dict["January"][1], self.format_date_to_test(c, c_round_2))
 
+    @staticmethod
+    def campaign_csv_columns():
```

Comment:
Columns are already defined in `plugins/polio/api/campaigns/campaigns.py`.

By redifining columns here, you are not testing `campaigns.campaign_csv_columns()` but `test_base.campaign_csv_columns()`.

You should use only one source in order to avoid desynchronization:

```python
from plugins.polio.api.campaigns.campaigns import CampaignViewSet

campaign_csv_columns = [col["title"] for col in CampaignViewSet.campaign_csv_columns()]
```